packages/components/src/spinners/CenteredSpinner.test.tsx

```diff
@@ -0,0 +1,46 @@
+import { withFakeTimers } from '@terra-ui-packages/test-utils';
+import { act, screen } from '@testing-library/react';
+import React from 'react';
+
+import { renderWithTheme } from '../internal/test-utils';
+import { visuallyHidden } from '../styles';
+import { CenteredSpinner } from './CenteredSpinner';
+
+describe('CenteredSpinner', () => {
+  it('renders a spinner icon', () => {
+    // Act
+    renderWithTheme(<CenteredSpinner />);
+
+    // Assert
+    const icon = document.querySelector('[data-icon="loadingSpinner"]');
+    expect(icon).toBeInTheDocument();
+  });
+
+  it(
+    'renders a visually hidden alert after a delay',
+    withFakeTimers(() => {
+      // Act
+      renderWithTheme(<CenteredSpinner message='Loading the data' />);
+
+      const isMessageRenderedImmediately = !!screen.queryByText('Loading the data');
+      act(() => jest.advanceTimersByTime(150));
+
+      // Assert
+      expect(isMessageRenderedImmediately).toBe(false);
+
+      const message = screen.getByText('Loading the data');
+      expect(message).toHaveAttribute('role', 'alert');
+      expect(message).toHaveStyle(visuallyHidden as Record<string, unknown>);
+    })
+  );
+
+  it('renders a spinner icon - no args', () => {
+    // Act
+    // call as raw function to ensure no args in call (React jsx version seems to not ensure this)
+    renderWithTheme(<div>{CenteredSpinner()}</div>);
```

Comment:
If `CenteredSpinner` is intended to be a component, it shouldn't be called like this. If it's intended to a render function, let's follow the convention of using a lower case name, preferably `render...`.

```suggestion
    renderWithTheme(<div><CenteredSpinner /></div>);
```

Reply:
the comment in the code explains why the test is calling it like this...  This call style lets us go from 50% branch coverage to 100% since it ensures the default {} empty props case is actually called in our unit tests.  I can make the comment more explicit to also say that this is NOT the preferred calling style.

Reply:
I don't think we need to test something that should never be used.